vector/array.go

```diff
@@ -0,0 +1,26 @@
+package vector
+
+import (
+	"github.com/brimdata/zed"
+)
+
+type Array struct {
+	mem
+	Typ     *zed.TypeArray //XXX type array or set
+	Lengths []int32
+	Values  Any
+}
+
+var _ Any = (*Array)(nil)
```

Comment:
This is an assertion that it fits the `Any` interface?

Reply:
Exactly.